draft-ietf-quic-transport.md

```diff
@@ -3509,7 +3509,7 @@ guidance offered below seeks to strike this balance.
 ### Sending ACK Frames {#sending-acknowledgements}
 
 Every packet SHOULD be acknowledged at least once, and ack-eliciting packets
-MUST be acknowledged at least once within the maximum ack delay. An endpoint
+MUST be acknowledged at least once within the maximum ACK delay. An endpoint
```

Comment:
This looks like the only place we use "ACK delay" as a concept.  We might be better referring to the max_ack_delay transport parameter.

Reply:
Will make the change